ui/src/components/header.tsx

```diff
@@ -222,6 +222,10 @@ export const Header = () => {
               </Link>
             )}
             <DropdownMenuSeparator />
+            <DropdownMenuItem className='flex gap-1' disabled>
+              <div>Logged in as:</div>
+              <div className='font-semibold'>{user.username}</div>
+            </DropdownMenuItem>
```

Comment:
```suggestion
            <DropdownMenuItem disabled>
              Logged in as: <span className='font-semibold'>{user.username}</span>
            </DropdownMenuItem>
```

Reply:
Something like this would probably be a better HTML.

Reply:
The current "Logged in as: username" also takes quite a lot more space than the other items, streching the menu. Would it work to display it in less space, maybe just the username, or put it in two rows? Check how other apps do it.

Reply:
I'd rather go for @sschuberth 's suggestion and just put the username to the top row ("header") of the menu.